standards/certification/pipeline.md

```diff
@@ -0,0 +1,218 @@
+   ```
+
+   Ideally, insert your subject so that the list (after `gx-scs`) remains sorted.
+
+3. Add your subject to [compliance-monitor/bootstrap.yaml](https://github.com/SovereignCloudStack/standards/blob/main/compliance-monitor/bootstrap.yaml).
+   This is necessary to that the reports will be accepted as genuine. Add a section like so:
+
+   ```diff
+      - subject: artcodix
+        delegates:
+          - zuul_ci
+   +  - subject: SUBJECT_NAME
+   +    delegates:
+   +      - zuul_ci
+   ```
+
+   Again, insert your subject so that the list (after `gx-scs`) remains sorted.
+
+4. Finally, add secrets to [.zuul.d/secure.yaml](https://github.com/SovereignCloudStack/standards/blob/main/.zuul.d/secure.yaml).
+   This is necessary so the tests can access your cloud.
+
+   This step is the most involved, and you can always have us do it for you; in that case, please send us
+   the application credential id and secret via an encrypted channel, e.g. Matrix.
+
+   To proceed, you need `zuul-client` installed:
+
+   ```shell
+   pipx install zuul-client
+   ```
+
+   Then you can execute:
+
+   ```shell
+   $ zuul-client --zuul-url https://zuul.sovereignit.cloud/ encrypt --tenant scs --project SovereignCloudStack/standards
+   <PASTE application credential id HERE>
+   <HIT ctrl-d>
+
+   ...
+    - secret:
+        name: <name>
+        data:
+          <fieldname>: !encrypted/pkcs1-oaep
+            - ...
+
+   $ zuul-client --zuul-url https://zuul.sovereignit.cloud/ encrypt --tenant scs --project SovereignCloudStack/standards
+   <PASTE application credential secret HERE>
+   <HIT ctrl-d>
+
+   ...
+    - secret:
+        name: <name>
+        data:
+          <fieldname>: !encrypted/pkcs1-oaep
+            - ...
+   ```
+
+   Copy the parts of the respective outputs starting in the final line shown here (the one starting `-`).
+   Insert them like so:
+
+   ```diff
+   +   SUBJECT_NAME_ac_id: !encrypted/pkcs1-oaep
+   +     - ENCRYPTED_ID
+   +   SUBJECT_NAME_ac_secret: !encrypted/pkcs1-oaep
+   +     - ENCRYPTED_SECRET
+   ```
+
+   Note that you have to use the same keys as in Step 3 (that is, with special characters replaced).
+
+5. Commit your changes and open a pull request:
+
+   ```shell
+   git commit -asm "Add SUBJECT_NAME"
+   git push  # the output of this command will show you the URL for creating the pull request
+   ```
+
+## How to feed the compliance monitor yourself
+
+Note: you may have to adapt these instructions to your infrastructure. For instance, the secrets
+we create here are stored locally. If you want to include the check suite into your own
+continuous-integration pipeline, you may want to use some dedicated credential store and mechanism for
+injecting secrets. You may want to take inspiration from our own Zuul setup by looking at
+[.zuul.d](https://github.com/SovereignCloudStack/standards/tree/main/.zuul.d) and
+[playbooks](https://github.com/SovereignCloudStack/standards/tree/main/playbooks).
```

Comment:
Do we want to mention that other job schedulers like Jenkins or even cron jobs can be used as well. This avoids scaring people that are overwhelmed by the capabilities that zuul offers.

Reply:
added language for that